addons/scoreboard/mergedplayer.lua

```diff
@@ -234,8 +234,21 @@ function MergedPlayer:wsavg()
     end
 end
 
-
-
+function MergedPlayer:wsacc()
+    local hits, misses = 0, 0
+    
+    for _, p in ipairs(self.players) do
```

Comment:
I'm not very familiar with *Scoreboard's* code, but this line looks weird. Wouldn't this sum up the stats from all players? Or what info does the `self.players` variable hold?

I can see that the above function also does this, so I won't go as far as call this wrong, but it's definitely not intuitive... any ideas?